### DISCUSSION TITLE
Is there any reason for version 5.3.7 not to have stub files generated?

### DISCUSSION THREAD

**Discussion by iamrafaelmelo** [phpredis/phpredis#2251] [Q&A]
We are using phpredis 5.3.7 in the company, and I am creating an ide helper for version 5.3.7 for now, I can get the constants and methods of the classes in the source code, but it would be of great help if I had the stubs file generated with the parameter names, fgiz some tests with the version 6 stubs, but it has some incompatible parameters. Would it be possible to generate these files? Could I be generating this myself?

I would love to help!

**Comment by michael-grunder** [comment]
Closing since we've got generated stubs for many versions.
